microsoft/testsuites/kvm/kvm_unit_tests_tool.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import string
+from pathlib import Path, PurePath
+from typing import List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.tools import Git, Make
+
+
+class KvmUnitTests(Tool):
+    # These tests take some time to finish executing. The default
+    # timeout of 600 is not sufficient.
```

Comment:
The timeout of test case is 600 seconds as well. It may need to extend in test case level too.